sandbox/grist/predicate_formula.py

```diff
@@ -50,7 +49,7 @@ def parse_predicate_formula(formula):
     return result
   except SyntaxError as e:
     # In case of an error, include line and offset.
-    raise SyntaxError("%s on line %s col %s" % (e.args[0], e.lineno, e.offset))
+    raise SyntaxError("%s on line %s col %s" % (e.args[0], e.lineno, e.offset)) from e
```

Comment:
Cool! I didn't know about `raise...from`!

Reply:
But I think it doesn't work in python 2, so we can't use it.

Reply:
Oops, that's sad... Using `six.reraise` instead.